contracts/hardhat.config.ts

```diff
@@ -0,0 +1,109 @@
+import { HardhatUserConfig, task } from "hardhat/config";
+import "@nomicfoundation/hardhat-toolbox";
+
+import "hardhat-abi-exporter";
+import "@solidstate/hardhat-bytecode-exporter";
+
+const abiExportPath = "./artifacts/abi/";
+const bytecodeExporterPath = "./artifacts/bin/"
+
+const config: HardhatUserConfig = {
+  solidity: {
+    version: "0.8.9",
+    settings: {
+      optimizer: {
+        enabled: true,
+        runs: 1000,
+      },
+      remappings : ["@openzeppelin=node_modules/@openzeppelin"],
+      outputSelection: { "*": { "*": [ "*" ], "": [ "*" ] } }
+    },
+  },
+  abiExporter : {
+    path: abiExportPath,
+    runOnCompile: true,
+    clear: true,
+    format: "json",
+  },
+  bytecodeExporter : {
+    path: bytecodeExporterPath,
+    runOnCompile: true,
+    clear: true,
+  }
+};
+
+import { TASK_COMPILE } from "hardhat/builtin-tasks/task-names";
```

Comment:
Can this task be in a different file ?

Reply:
https://github.com/obscuronet/go-obscuro/pull/966 - it is moved in the following PR, as I kinda fudged it up.